src/core/agent/services/signifyNotificationService.test.ts

```diff
@@ -493,4 +493,55 @@ describe("Signify notification service of agent", () => {
     );
     expect(markNotificationMock).toBeCalledWith(notification[0].i);
   });
+
+  test("Should skip if notification route is /multisig/exn", async () => {
```

Comment:
This is a misleading test title. It should skip IF it's not `/ipex/admit`, OR the identifier is missing OR the credential exists. Please cover the cases properly with proper titles